### PR TITLE
add parameter to read out single value from channel

### PR DESCRIPTION
**Pull request by peendebak**
Add parameter to read out the digitizer channels as a simple instrument

@lukabavdaz @giulioungaretti @jenshnielsen @WilliamHPNielsen

### CODE REVIEW

File: qcodes/instrument_drivers/Spectrum/M4i.py [hunk 1]
```diff
@@ -568,15 +573,38 @@ def convert_to_voltage(self, data, input_range):
         resolution = self.ADC_to_voltage()
         return data * input_range / resolution
 
-    def set_channel_settings(self, i, mV_range, input_path, termination, coupling, compensation):
+    def initialize_channels(self, channels=None, mV_range=1000, input_path=0, termination=0, coupling=0, compensation=None):
+        """ Setup channels of the digitizer for simple readout using Parameters
+
+        Args:
```
Comment (WilliamHPNielsen) [inline qcodes/instrument_drivers/Spectrum/M4i.py]: The docstring should document all arguments.

Comment (eendebakpt) [inline qcodes/instrument_drivers/Spectrum/M4i.py]: @WilliamHPNielsen Updated to docstring. I refer to another function now.

**Comment by peendebak** [comment]
@giulioungaretti @WilliamHPNielsen Autopep should be fixed now.

**Comment by WilliamHPNielsen** [comment]
I don't have a good overview of this driver, but the changes in this PR seem fine.

**Pull request merged**